pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "jupyter_smart_on_fhir"
+version = "0.1.0"
+
+[tool.setuptools]
+packages = ["hub_service", "server_extension"]
```

Comment:
Can this be a single `jupyter_smart_on_fhir` package, with the two implementations in submodules? i.e. `jupyterhub_smart_on_fhir/hub_service.py`, `jupyterhub_smart_on_fhir/server_extension.py`?